### ISSUE TITLE
Support org.bluez.LEAdvertisement1 interface

### ISSUE THREAD

**Issue by robert-ancell** [canonical/bluez.dart#40]
https://github.com/bluez/bluez/blob/master/doc/advertising-api.txt

**Comment by RossComputerGuy** [comment]
Since the link is a 404, [here is a viewable version](https://github.com/bluez/bluez/blob/5.70/doc/advertising-api.txt).

**Comment by anderspitman** [comment]
@RossComputerGuy looks like this was fixed by #96, correct?

**Comment by RossComputerGuy** [comment]
Yes, I think this was just forgotten to be closed unless @robert-ancell knows about anything else which should be implemented.
